example_site/src/components/ConfigurationControls.js

```diff
@@ -15,7 +15,7 @@ import { partnerPresets, partnerConfigurations } from "./constants";
  */
 export default function ConfigurationControls({ isOpen }) {
   const { register, watch, handleSubmit } = useForm();
-  const [partner, setPartner] = useState('default');
+  const [partner, setPartner] = useState('generic');
```

Comment:
Given the discussion w/ Alex and that we're trying to move away from partner-specific jargon toward a flow-based experience, what do you think of updating this to `[siteConfiguration, setSiteConfiguration]`?

Reply:
In turn that would mean updating the names of the `partnerPresets` etc.